src/main/java/me/totalfreedom/totalfreedommod/TotalFreedomMod.java

```diff
@@ -296,83 +325,100 @@ public Initializer()
+            eventBlocker = new EventBlocker();
+            blockBlocker = new BlockBlocker();
+            mobBlocker = new MobBlocker();
+            interactBlocker = new InteractBlocker();
+            potionBlocker = new PotionBlocker();
+            loginProcess = new LoginProcess();
+            antiNuke = new AntiNuke();
+            antiSpam = new AntiSpam();
+            playerList = new PlayerList();
+            shop = new Shop();
+            votifier = new Votifier();
+            announcer = new Announcer();
+            chatManager = new ChatHandler();
+            discord = new Discord();
+            punishmentList = new PunishmentList();
+            banManager = new BanManager();
+            indefiniteBanList = new IndefiniteBanList();
+            permissionManager = new PermissionManager();
+            signBlocker = new SignBlocker();
+            entityWiper = new EntityWiper();
+            sitter = new Sitter();
+            vanishHandler = new VanishHandler();
+            pterodactyl = new Pterodactyl();
+
+            if (isPluginPresent("WorldGuard"))
+                worldRestrictions = new WorldRestrictions();
         }
 
         private void initAdminUtils()
         {
             // Single admin utils
-            cs = new CommandSpy();
-            ca = new Cager();
-            fm = new Freezer();
-            or = new Orbiter();
-            mu = new Muter();
-            ebl = new EditBlocker();
-            pbl = new PVPBlocker();
-            fo = new Fuckoff();
-            ak = new AutoKick();
-            ae = new AutoEject();
-            mo = new Monitors();
+            commandSpy = new CommandSpy();
+            cager = new Cager();
+            freezer = new Freezer();
+            orbiter = new Orbiter();
+            muter = new Muter();
+            editBlocker = new EditBlocker();
+            pvpBlocker = new PVPBlocker();
+            fuckoff = new Fuckoff();
+            autoKick = new AutoKick();
+            autoEject = new AutoEject();
+            monitors = new Monitors();
         }
 
         private void initBridges()
         {
             // Start bridges
-            btb = new BukkitTelnetBridge();
-            cpb = new CoreProtectBridge();
-            esb = new EssentialsBridge();
-            ldb = new LibsDisguisesBridge();
-            tfg = new TFGuildsBridge();
-            web = new WorldEditBridge();
-            wgb = new WorldGuardBridge();
+            if (isPluginPresent("BukkitTelnet"))
+                bukkitTelnetBridge = new BukkitTelnetBridge();
+            if (isPluginPresent("CoreProtect"))
+                coreProtectBridge = new CoreProtectBridge();
+            if (isPluginPresent("WorldGuard"))
+                worldGuardBridge = new WorldGuardBridge();
+            if (isPluginPresent("WorldEdit"))
+                worldEditBridge = new WorldEditBridge();
+
+            essentialsBridge = new EssentialsBridge();
```

Comment:
Why aren't these included in the if logic? Not sure having half of them enabled and half not is a good idea. Especially Libs & TFGuilds where they are not "Mandatory" plugins.